vcr-workflow.Rmd

```diff
@@ -0,0 +1,66 @@
+# vcr workflows {#vcr-workflows}
```

Comment:
why is this not a vignette with an Rmd fragment?

Reply:
can do